tasks/queue-granules/index.js

```diff
@@ -36,8 +35,7 @@ async function fetchGranuleProvider(prefix, providerId) {
 function groupAndBatchGranules(granules, batchSize) {
   const filteredBatchSize = isNumber(batchSize) ? batchSize : 1;
   const granulesByCollectionMap = groupBy(
-    granules,
-    (g) => constructCollectionId(g.dataType, g.version)
+    granules, granules.collectionId
```

Comment:
This is going to break discover granules workflows - if Queue Granules is used before Sync Granules then the output *does not* provide collectionId.   We cannot rely on it, despite it being part of the post-sync workflow.

Reply:
I believe we want it to work for *both*.